geonode_mapstore_client/client/js/utils/ResourceUtils.js

```diff
@@ -573,16 +570,11 @@ export function toMapStoreMapConfig(resource, baseConfig) {
         .map((layer) => {
             const mapLayer = maplayers.find(mLayer => layer.id !== undefined && mLayer?.extra_params?.msId === layer.id);
             if (mapLayer) {
-                const mapLayerDatasetStyles = layer.type === 'wms' ? cleanStyles([
-                    ...(mapLayer?.dataset?.defaul_style ? [mapLayer.dataset.defaul_style] : []),
-                    ...(mapLayer?.dataset?.styles || [])
-                ]).map(({ name }) => name) : [];
-                const template = mapLayer?.dataset?.featureinfo_custom_template || '';
+                const template = layer?.featureInfo?.template ?? mapLayer?.dataset?.featureinfo_custom_template ?? '';
```

Comment:
We should completely remove the featureInfo override here, including the template, only the style will be synchronized